spec/services/articles/feeds/variant_query_spec.rb

```diff
@@ -2,34 +2,35 @@
 
 RSpec.describe Articles::Feeds::VariantQuery, type: :service do
   # We're exercising each named feed variant to ensure that the queries are valid SQL.
-  describe ".build_for" do
-    Rails.root.glob(File.join(Articles::Feeds::VariantAssembler::DIRECTORY, "/*.json")).each do |pathname|
-      variant = pathname.basename(".json").to_s.to_sym
+  Rails.root.glob(File.join(Articles::Feeds::VariantAssembler::DIRECTORY, "/*.json")).each do |pathname|
+    variant = pathname.basename(".json").to_s.to_sym
+
+    describe ".build_for with #{variant} variant" do
       subject(:query_call) { variant_query.call }
 
       let(:variant_query) {  described_class.build_for(variant: variant, user: user) }
 
-      context "for #call with #{variant.inspect} and user is nil" do
+      describe "#call with nil user" do
         let(:user) { nil }
 
-        it "is a valid ActiveRecord::Relation" do
+        it "is a valid ActiveRecord::Relation", aggregate_failures: true do
           article = create(:article)
           expect(query_call).to be_a(ActiveRecord::Relation)
           expect(query_call.to_a).to match_array(article)
         end
       end
 
-      context "for #call with #{variant.inspect} and a non-nil user" do
+      describe "#call with a non-nil user" do
         let(:user) { create(:user) }
 
-        it "is a valid ActiveRecord::Relation" do
+        it "is a valid ActiveRecord::Relation", aggregate_failures: true do
```

Comment:
Quick, non-blocking suggestion (same as above!): can we trim this down to the following?
```suggestion
        it "is a valid ActiveRecord::Relation", :aggregate_failures do
```